vtuber/verilog/dtype/varray_test.cpp

```diff
@@ -84,13 +85,4 @@ TEST(TestVerilogArray, DISABLED_Unpack) {
 		EXPECT_EQ(a32_8[6].value(), 0xF9ADDDCF);
 		EXPECT_EQ(a32_8[7].value(), 0xA45F1DFE);
 	}
-	{
-		varray<vuint<64>, 5> a64_10;
```

Comment:
Why this being removed?

Reply:
This test is being removed becuase its bit width are mismatched. 
I modified it with correct bit-width.